scripts/lint-cli-startup-flow.py

```diff
@@ -8,6 +8,7 @@
 # startup performance
 STDLIBS_TO_PRELOAD = [
     "argparse",
+    "argcomplete",
```

Comment:
This is not a Python standard library, so it doesn't belong here. Remove it and maintain the allowlist instead.

Reply:
> This is not a Python standard library, so it doesn't belong here. Remove it and maintain the allowlist instead.

move it to allow list, and add stdlib shlex and subprocess as argcomplete needs them (show reason in the comment)